src/main/resources/application.yml

```diff
@@ -74,3 +74,6 @@ management:
     info:
       cache:
         time-to-live: 2000ms
+
+calculation:
+  version:  1
```

Comment:
We should probably put a note in the README about when to change this, whether we are using semver or whole numbers etc